cmd/id.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+	"github.com/trento-project/agent/internal/agent"
+)
+
+func NewIDCmd() *cobra.Command {
+	idCmd := &cobra.Command{ //nolint
+		Use:   "id",
+		Short: "Print the Agent identifier",
```

Comment:
We have the agent word already in small case in other help strings. I would continue using that way